client/resources/aid_desfire.json

```diff
@@ -371,127 +515,191 @@
-    },
     {
         "AID": "534531",
-        "Vendor": "Transport for New South Wales [TfNSW]",
+        "Vendor": "Transport for New South Wales (TfNSW)",
         "Country": "AU",
-        "Name": "Opal Card [SYD]",
+        "Name": "Opal Card (SYD)",
         "Description": "FIDs 00-06: Standard Data; 07: Card Balance/Number and Trip History",
         "Type": "transport"
     },
     {
         "AID": "554000",
         "Vendor": "Auckland Transport",
         "Country": "NZ",
-        "Name": "AT HOP Card [AKL]",
+        "Name": "AT HOP Card (AKL)",
         "Description": "FIDs: 00: Backup Data; 08/09/0A",
         "Type": "transport"
     },
     {
         "AID": "578000",
-        "Vendor": "Norwegian Public Roads Administration [NPRA]",
+        "Vendor": "Norwegian Public Roads Administration (NPRA)",
         "Country": "NO",
-        "Name": "NORTIC Transport",
+        "Name": "NORTIC (Norway Public Transport Card)",
         "Description": "Norwegian Ticketing Interoperable Concept // FID 0C: Card Issuer Header",
         "Type": "transport"
     },
     {
         "AID": "578001",
-        "Vendor": "Norwegian Public Roads Administration [NPRA]",
+        "Vendor": "Norwegian Public Roads Administration (NPRA)",
         "Country": "NO",
-        "Name": "NORTIC Transport",
+        "Name": "NORTIC (Norway Public Transport Card)",
         "Description": "FIDs 01: Product Retailer; 02: Service Provider; 03: Special Event; 04: Stored Value; 05: General Event Log; 06: SV Reload Log; 0A: Environment; 0C: Card Holder",
         "Type": "transport"
     },
     {
         "AID": "784000",
-        "Vendor": "Roads & Transport Authority [Government of Dubai]",
+        "Vendor": "Roads & Transport Authority (Government of Dubai)",
         "Country": "AE",
-        "Name": "nol Card [DXB]",
+        "Name": "nol Card (DXB)",
         "Description": "DXB nol Card",
         "Type": "transport"
     },
     {
         "AID": "A00216",
         "Vendor": "ITSO Ltd",
         "Country": "UK",
-        "Name": "ITSO",
-        "Description": "Appears to be used across UK Transit Agencies except LHR Oyster.",
+        "Name": "EMR Smartcard (EMA) / GWR touch (BRS)",
+        "Description": "ITSO Public Transport Application // Provision of Citizen Services #0",
         "Type": "transport"
     },
     {
         "AID": "EF2011",
-        "Vendor": "Helsinki Region Transport [HRT]",
+        "Vendor": "Helsinki Region Transport (HRT)",
         "Country": "FI",
-        "Name": "HSL Card [HEL/TLL/TAY]",
+        "Name": "HSL Card (HEL/TLL/TAY)",
         "Description": "HEL/TLL/TAY HSL Card",
         "Type": "transport"
     },
     {
         "AID": "F001D0",
         "Vendor": "Arabako Foru Aldundia",
         "Country": "ES",
-        "Name": "BAT Card [VIT]",
-        "Description": "VIT BAT Card",
+        "Name": "BAT Card (VIT)",
+        "Description": "VIT BAT Card // Used on Euskotren // Interoperable PT Card",
```

Comment:
Not sure that JSON - format allows for inline comments.  

You sure this is correct JSON?